Index: app-cc/cc-test/gtest/gw/api/policy/refresh/compare/PolicyDifferenceTest.gs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app-cc/cc-test/gtest/gw/api/policy/refresh/compare/PolicyDifferenceTest.gs	(revision feb3d39c5b75c4b469e4cf3db434c13661581cff)
+++ app-cc/cc-test/gtest/gw/api/policy/refresh/compare/PolicyDifferenceTest.gs	(revision )
@@ -10,6 +10,7 @@
 uses gw.api.testdata.TestDataUtil
 uses gw.api.databuilder.EndorsementBuilder
 uses gw.multiapp.LatestMultiappVersionUtil
+uses gw.plugin.policy.impl.PolicySearchPolicyGenerator
 uses gw.testharness.v3.CCServerTestV3
 uses gw.testharness.v3.CCSuiteNamesV3
 uses gw.testharness.v3.Suites
@@ -182,6 +183,10 @@
                   .withDeductibleAndCovTerm(new BigDecimal(10000).ofDefaultCurrency())))
         .withContactInRole(company, TC_INSURED)
         .withContactInRole(agent, TC_AGENT)
+  }
+
+  function testDataLoad() {
+    new PolicySearchPolicyGenerator().generatePolicies()
   }
 
   /**
Index: app-cc/cc/config/locale/display_en_US.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app-cc/cc/config/locale/display_en_US.properties	(revision feb3d39c5b75c4b469e4cf3db434c13661581cff)
+++ app-cc/cc/config/locale/display_en_US.properties	(revision )
@@ -13216,6 +13216,7 @@
 Web.Tools.CCSampleData.CatastropheSearchGroupName = Catastrophe Search
 Web.Tools.CCSampleData.DataSetImportSucceeded = Import succeeded for {0} (time {1})
 Web.Tools.CCSampleData.DemoGroupDescription = Demo sample data, for product demos
+Web.Tools.CCSampleData.DemoNonAdminGroupDescription = Demo sample data without Admin data, for product demos (requires 'Admin' loaded first)
 Web.Tools.CCSampleData.DemoGroupName = Demo
 Web.Tools.CCSampleData.DurationFormat = {0}\:{1}\:{2}
 Web.Tools.CCSampleData.ExchangeRateGroupDescription = One set of market exchange rates
Index: app-cc/cc-gunit-content/gsrc/gw/sampledata/SampleDataGroup.gs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app-cc/cc-gunit-content/gsrc/gw/sampledata/SampleDataGroup.gs	(revision feb3d39c5b75c4b469e4cf3db434c13661581cff)
+++ app-cc/cc-gunit-content/gsrc/gw/sampledata/SampleDataGroup.gs	(revision )
@@ -92,6 +92,80 @@
     \ data, bundle -> data.demoSampleData(bundle)
   ),
 
+  ClaimsOnly(
+    true,
+    \ -> "  Demo, no admin",
+    \ -> DisplayKey.get("Web.Tools.CCSampleData.DemoNonAdminGroupDescription"),
+      {
+          SampleClaims,
+          SampleAggregateLimits,
+          SampleAuthorityLimits,
+          SampleMessages,
+          SampleContacts
+      },
+    \ data, bundle -> data.demoSampleData(bundle)
+  ),
+
+  PAClaimsOnly(
+    true,
+    \ -> "SamplePersonalAutoClaims",
+    \ -> DisplayKey.get("Web.Tools.CCSampleData.DemoNonAdminGroupDescription"),
+      {
+          SamplePersonalAutoClaims
+      },
+    \ data, bundle -> data.demoSampleData(bundle)
+  ),
+
+  CAClaimsOnly(
+    true,
+    \ -> "SampleCommercialAutoClaims",
+    \ -> DisplayKey.get("Web.Tools.CCSampleData.DemoNonAdminGroupDescription"),
+      {
+          SampleCommercialAutoClaims
+      },
+    \ data, bundle -> data.demoSampleData(bundle)
+  ),
+
+  CPClaimsOnly(
+    true,
+    \ -> "SampleCommercialPropertyClaims",
+    \ -> DisplayKey.get("Web.Tools.CCSampleData.DemoNonAdminGroupDescription"),
+      {
+          SampleCommercialPropertyClaims
+      },
+    \ data, bundle -> data.demoSampleData(bundle)
+  ),
+
+  TravelClaimsOnly(
+    true,
+    \ -> "SampleTravelClaims Claims",
+    \ -> DisplayKey.get("Web.Tools.CCSampleData.DemoNonAdminGroupDescription"),
+      {
+          SampleTravelClaims
+      },
+    \ data, bundle -> data.demoSampleData(bundle)
+  ),
+
+  WCClaimsOnly(
+    true,
+    \ -> "SampleWorkersCompClaims Claims",
+    \ -> DisplayKey.get("Web.Tools.CCSampleData.DemoNonAdminGroupDescription"),
+      {
+          SampleWorkersCompClaims
+      },
+    \ data, bundle -> data.demoSampleData(bundle)
+  ),
+
+  GLClaimsOnly(
+    true,
+    \ -> "SampleGeneralLiabilityClaims Claims",
+    \ -> DisplayKey.get("Web.Tools.CCSampleData.DemoNonAdminGroupDescription"),
+      {
+          SampleGeneralLiabilityClaims
+      },
+    \ data, bundle -> data.demoSampleData(bundle)
+  ),
+
   CatastropheSearch(
     true,
     \ -> DisplayKey.get("Web.Tools.CCSampleData.CatastropheSearchGroupName"),
